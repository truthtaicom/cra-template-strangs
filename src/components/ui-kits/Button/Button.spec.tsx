import React from 'react'
import renderer from 'react-test-renderer'
import Button from './Button'

test('AppTitle', () => {
  const component = renderer.create(<Button />)
  const tree = component.toJSON()

  expect(tree).toMatchSnapshot()
})
