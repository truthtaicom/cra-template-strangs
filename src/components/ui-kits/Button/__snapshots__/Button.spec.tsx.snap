// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppTitle 1`] = `
<button
  className="sc-AxjAm bDnbtj"
/>
`;
